### PR TITLE
Final polishing for Azure.ResourceManager.AppService for GA

### PR DESCRIPTION
**Pull request by ArthurMa1978**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs [hunk 1]
```diff
@@ -617,7 +617,7 @@ public partial class AppServicePlanData : Azure.ResourceManager.Models.TrackedRe
         public string ResourceGroup { get { throw null; } }
         public Azure.ResourceManager.AppService.Models.AppServiceSkuDescription Sku { get { throw null; } set { } }
         public System.DateTimeOffset? SpotExpireOn { get { throw null; } set { } }
-        public Azure.ResourceManager.AppService.Models.StatusOption? Status { get { throw null; } }
+        public Azure.ResourceManager.AppService.Models.AppServicePlanStatusOption? Status { get { throw null; } }
```
Comment (Yao725) [inline sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs:620]: I think maybe we can drop the Option suffix here.

File: sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs [hunk 2]
```diff
@@ -709,22 +709,22 @@ public partial class AppServicePlanVirtualNetworkConnectionGatewayResource : Azu
...
         public virtual bool HasData { get { throw null; } }
-        public virtual Azure.Response<Azure.ResourceManager.AppService.Models.AppServiceVnetRoute> CreateOrUpdateVnetRoute(string routeName, Azure.ResourceManager.AppService.Models.AppServiceVnetRoute route, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.AppService.Models.AppServiceVnetRoute>> CreateOrUpdateVnetRouteAsync(string routeName, Azure.ResourceManager.AppService.Models.AppServiceVnetRoute route, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.AppService.Models.AppServiceVirtualNetworkRout> CreateOrUpdateVnetRoute(string routeName, Azure.ResourceManager.AppService.Models.AppServiceVirtualNetworkRout route, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (Yao725) [inline sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs:726]: Did we miss the letter `e` here? I guess it should be `AppServiceVirtualNetworkRoute`.

**Review by Yao725 — APPROVED** [review]

File: sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs:726]: fixed

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**